app/assets/stylesheets/theme_previews.css

```diff
@@ -2,7 +2,6 @@
 /* theme selector for user theme selection form */
 .theme-previews {
   display: grid;
-  grid-template-columns: repeat(2, 1fr); /* Two columns layout */
```

Comment:
in the themese form - this will make it a single column selector as oppose to a 2 x 2 grid system

Reply:
I like it, but when I scroll down, it looks weird when the part of the navbar above the save themes button is transparent.

The other thing is that because you're scrolling through instead of being able to view the whole resume theme on one page, it makes it hard to see what the theme as a whole is (maybe instead make a horizontal carousel of resume themes?). Also, to navigate to a specific form requires a lot of scrolling.

Reply:
yea great idea - horizontal carousel resonates with me a lot and since we aren't expecting mobile responsive users on this page it could be a quick win